scripts/fetch_trending.py

```
@@ -0,0 +1,234 @@
+
+        if not stable_release or not stable_release.get('assets'):
+            return False
+
+        # Check for relevant installer files
+        extensions = PLATFORMS[platform]['installer_extensions']
+        for asset in stable_release['assets']:
+            asset_name = asset['name'].lower()
+            if any(asset_name.endswith(ext) for ext in extensions):
+                return True
+
+        return False
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)